docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM dealii/dealii:v8.5.pre.4-gcc-mpi-fulldepsmanual-debugrelease
+
+LABEL maintainer <rene.gassmoeller@mailbox.org>
+
+# Build aspect
+RUN git clone https://github.com/geodynamics/aspect.git ./aspect && \ 
+    mkdir aspect/build && \
+    cd aspect/build && \
+    cmake -DCMAKE_BUILD_TYPE=Release \
+          -DDEAL_II_DIR=$HOME/deal.II-install \
+          .. && \
+    make -j4 >log 2>&1 && \
+    mv aspect $HOME/aspect/ && \
```

Comment:
I redirected the output, because gcc 6.2 currently creates hundreds of deprecation warnings while compiling aspect (if I remember correctly there was an issue within gcc, that should be fixed in newer versions), so the output is very confusing. I am fine with removing this when this is fixed at some point. Anyway, I think most users will not need to build the image themselves, because it is available at docker hub.